src/components/oInput.js

```diff
@@ -67,14 +76,21 @@ oInput.prototype.dbIndex = function(index) {
 }
 
 oInput.prototype.disabled = function(required) {
-  if(typeof disabled !== 'boolean') return this;
+  if(disabled !== 'true' && disabled !== 'false' && typeof disabled !== 'boolean') return this;
 
   this.element.attr({
     disabled
   });
   return this;
 }
```

Comment:
Drobny babol się wkradł :D required -> disabled
Proponuje też zrobić, że jeśli to nie jest boolean i jeśli jest typu string to disabled.toLowerCase() i wtedy przyrównać do true i false. Będzie większa elastyczność, że będą mogli sobie wpisac "True" też. Jak już chcemy, żeby stringiem też przeszło

Reply:
Babol poprawiony :D Pozostałe - odrzuciliśmy